lnx-server/src/routes/index.rs

```diff
@@ -307,3 +307,19 @@ pub async fn clear_documents(req: LnxRequest) -> LnxResponse {
 
     json_response(200, "changes registered")
 }
+
+pub async fn get_index_stats(req: LnxRequest) -> LnxResponse {
+    let state = req.data::<State>().expect("get state");
+    let index = get_or_400!(req.param("index"));
+    let index: Index = get_or_400!(state.engine.get_index(index));
+
+    let stats = index.get_doc_count().await?;
+
+    json_response(
+        200,
+        &serde_json::json!({
+            "num_docs": stats.docs,
+            "num_deleted_docs": stats.deleted_docs,
+        }),
+    )
```

Comment:
It might be easier to derive Serialize on the stats struct instead, that way it does not require the additional `json!` macro stuff.

Reply:
True. I changed to serialize `IndexStats` directly now.